API-doc-FORD-file.md

```diff
@@ -9,6 +9,9 @@ media_dir: doc/media
 fpp_extensions: fypp
 preprocess: true
 macro: MAXRANK=3
+       PROJECT_VERSION_MAJOR=0
+       PROJECT_VERSION_MINOR=0
+       PROJECT_VERSION_PATCH=0
```

Comment:
```suggestion
       PROJECT_VERSION_MAJOR=0
       PROJECT_VERSION_MINOR=1
       PROJECT_VERSION_PATCH=0
```

Reply:
Or is this overridden during the fypp processing?

Reply:
This is supposed to be a stub, similar like the `MAXRANK=3` argument above, which would would not allow to actually compile stdlib.

Reply:
Don't these macros get applied when FORD runs fypp to preprocess the sources for the documentation? At least that's what I understand from here: https://github.com/Fortran-FOSS-Programmers/ford/wiki/Project-File-Options

Reply:
It does show up in the final docs for the compile time constant:

![image](https://user-images.githubusercontent.com/28669218/143597723-ce163cf4-1a13-45fa-a397-40b2ec8c7f54.png)

Otherwise the value set won't affect the output of the documentation. I would prefer to leave this as a stub, so we don't have to worry about it becoming outdated when we release a new version.

Reply:
Comments are not allowed in the FORD meta-data section. Is there somewhere we could document these values are stubs, and don't reflect the actual library?